lib/plugins/aws/package/compile/events/websockets/lib/api.test.js

```diff
@@ -86,7 +86,35 @@ describe('#compileApi()', () => {
           ],
         },
       });
-    }));
+    });
```

Comment:
Since the two items above were missing, the closing braces and parentheses were incorrect.